ckcp/run.sh

```diff
@@ -97,34 +85,21 @@ else
         kubectl delete pods -l kcp.dev/cluster=local --field-selector=status.phase==Succeeded -n kcpe2cca7df639571aaea31e2a733771938dc381f7762ff7a077100ffad;
       fi;
 
-      #install namespaces in ckcp
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl create namespace default
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl create namespace tekton-pipelines
-
-      #install pipelines CRDs in ckcp
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-pipelinerun.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-taskrun.yaml
-
-      # will go away with v1 graduation
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-run.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-resource.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-condition.yaml
-
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply $(ls pipeline/config/config-* | awk ' { print " -f " $1 } ')
+      KUBECONFIG=$KUBECONFIG_KCP kubectl apply -k ../gitops/tekton-pipeline/overlays/patched
```

Comment:
Use SCRIPT_DIR intead of relative path (everywhere).

Reply:
Already did that. It was kubeconfig/admin.kubeconfig earlier everywhere.. I changed that to $KUBECONFIG_KCP

Reply:
I mean instead of ../gitops/.... use $SCRIPT_DIR/gitops

Reply:
Done. Fixed all the paths.